python/hail/methods/impex.py

```diff
@@ -838,6 +840,11 @@ def import_bgen(path,
 
     **Row Fields**
 
+    Between two and four row fields are created. The `locus` and `alleles` are
```

Comment:
I kinda don't want this to be user-exposed, but hidden in the backend. When the pruner goes in it'll be possible for us to do this automatically.

Reply:
Agreed, but until then this is the only way to get Caitlin (and others) cooking.

Reply:
I would prefer to leave it as an underscored parameter, and undocumented. Would that be OK With you? Otherwise, removing this becomes a breaking change.